src/printer.js

```diff
@@ -477,7 +477,10 @@ function printExpression(path, options, print) {
         // @TODO: for now just reusing double/single quote preference from doc. could eventually
         // use setting from options. need to figure out how this works w/ complex strings and interpolation
         // also need to figure out splitting long strings
-        const quote = node.isDoubleQuote ? '"' : "'";
+        let quote = node.isDoubleQuote ? '"' : "'";
+        if (path.getParentNode().kind === "encapsed") {
+          quote = "";
```

Comment:
How is this working? I've just looked at the implementation of `makeString`, but I don't see how passing an empty quote is affecting the escaping ...

Reply:
I don't think it would affect the escaping - but when the string is part of an encapsed node, that parent node is what's responsible for printing the actual quote. so if you had `$test = "test ${string}";` - that actually gets broken up into multiple strings. the parent node is an encapsed node (which is responsible for printing the quote), and the children are `test ` and `${string}` i think - which we wouldn't want to re-print quotes for

Reply:
Right - and is the change affecting the tests in any way?

Reply:
I don't think so? Other than keeping escaped line breaks escaped (which we're not currently doing)